### DISCUSSION TITLE
Restoring my deleted repository

### DISCUSSION THREAD

**Discussion by bhavishbsshetty2000** [community/community#22491] [New to GitHub]
<p>I deleted my repository by mistake.Is there any way to restore it?<br>
I cant find it in deleted repository.Does it take time to be displayed?</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>check this out,</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://docs.github.com/en/repositories/creating-and-managing-repositories/restoring-a-deleted-repository">
  <header class="source">
      <img src="https://docs.github.com/assets/cb-803/images/site/favicon.svg" class="site-icon" width="16" height="16">

      <a href="https://docs.github.com/en/repositories/creating-and-managing-repositories/restoring-a-deleted-repository" target="_blank" rel="noopener">GitHub Docs</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/47092464/181071436-ad85964d-fd53-4d99-84e0-aa09edc3fc05.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png">

<h3><a href="https://docs.github.com/en/repositories/creating-and-managing-repositories/restoring-a-deleted-repository" target="_blank" rel="noopener">Restoring a deleted repository - GitHub Docs</a></h3>

  <p>You can restore some deleted repositories to recover their contents.</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

**Comment by jennypherx** [comment]
I just had the same issue, were you able to restore your repo?
Mine is not showing up in the "deleted repository" either.

**Comment by appjedi** [comment]
I have an organization account and don't see the "Deleted Repository" under archive in left side.  Any ideas?
